tests/test_method_equivalence.py

```diff
@@ -29,6 +29,7 @@ class Datum:
     Y = np.sin(X) + 0.9 * np.cos(X * 1.6) + rng.randn(*X.shape) * 0.8
     Y = np.tile(Y, 2)  # two identical columns
     Xtest = rng.rand(10, 1) * 10
+    data = X, Y
```

Comment:
I don't have any strong preference between `data = X, Y` as here or `data = (X, Y)` as below, but would be good to be consistent!

Reply:
```suggestion
    data = (X, Y)
```